wgsl/extract-grammar.py

```diff
@@ -529,7 +529,7 @@ def not_token_only(value):
     grammar_package.write('        "nan": "^2.15.0"\n')
     grammar_package.write('    },\n')
     grammar_package.write('    "devDependencies": {\n')
-    grammar_package.write('        "tree-sitter-cli": "^0.20.0"\n')
+    grammar_package.write('        "tree-sitter-cli": "^0.19.0"\n')
```

Comment:
This is changed to 0.19.0 but the install-dependencies.sh is changed to 0.20.0. Shouldn't they be consistent?

Reply:
Please check the consecutive PR. tree-sitter team issued a fix for this situation.

Reply:
@kainino0x this is the PR: https://github.com/gpuweb/gpuweb/pull/3416

And the tree-sitter fix's source discussion: https://github.com/tree-sitter/py-tree-sitter/issues/60#issuecomment-1237436565

Reply:
ah I see, the two packages aren't kept exactly in line. Thanks for the pointers!